src/entities/AppUser.ts

```diff
@@ -45,4 +45,14 @@ export class AppUser {
     default: AppUserRole.GUEST,
   })
   role: string;
+
+  @Column({ nullable: true, type: 'json' })
+  availabilities: Availabilities;
+}
+
+export type Availabilities = TimeRange[];
+
+interface TimeRange {
```

Comment:
Put interface definition to the top of the file to conform to convention of our codebase